src/Components/Publishing/__tests__/Article.test.tsx

```diff
@@ -43,3 +44,42 @@ it("renders news articles in news layout", () => {
   const article = mount(<Article article={NewsArticle} />)
   expect(article.find(NewsLayout).length).toBe(1)
 })
+
+it("renders mobile MinimalCtaBanner for standard article layouts", () => {
+  const article = shallow(
```

Comment:
Why are we checking for standard layout, but not other layouts? Should at least check that it isn't appearing on series.